contracts/CurrencyNetworkRegistry.sol

```diff
@@ -0,0 +1,95 @@
+pragma solidity ^0.5.8;
+
+import "./CurrencyNetworkInterface.sol";
+import "./lib/ERC165Query.sol";
+
+
+contract CurrencyNetworkRegistry is ERC165Query {
+
+    struct CurrencyNetworkMetadata {
+        address author;
```

Comment:
I would rename `author` to `registeredBy`, not to mix it up with the `author` of the currency network as defined by the one deploying the currency network.

Reply:
Good point. Done.